pkgs/flox/default.nix

```diff
@@ -197,9 +195,7 @@ in
       passthru.envs = envs;
       passthru.manpages = manpages;
       passthru.rustPlatform = rustPlatform;
-      passthru.flox-bash = flox-bash;
       passthru.cargoDeps = cargoDepsArtifacts;
-      passthru.nixPatched = flox-bash.passthru.nixPatched;
```

Comment:
Keep this please for now. Pass along whichever Nix should be in the closure.

Reply:
Oh I reread your comment I think I know what you mean now. Fixing it.